polkadot/runtime/parachains/src/paras/mod.rs

```diff
@@ -653,6 +663,20 @@ pub mod pallet {
 		///
 		/// TODO: Remove once coretime is the standard across all chains.
 		type AssignCoretime: AssignCoretime;
+
+		/// The fungible instance used by the runtime.
+		type Fungible: Mutate<Self::AccountId, Balance: From<BlockNumberFor<Self>>>;
```

Comment:
something to be aware of: we're adding yet another dependency on balances here, which will make it harder to remove them from the relay chain. i guess all of this can be replaced by a credit system akin to the coretime credits. some future work :)

Reply:
Yeah that is true, but I don't see the balances support getting removed in the near future.